### ISSUE TITLE
[VT-36] vit; command history (up/ down arrow keys)

### ISSUE THREAD

**Issue by taskwarrior** [scottkosty/vit#58]
_David Patrick on 2013-10-05T03:27:35Z says:_ 

 When at the command-prompt in vit, if a command history could be accessed with the up and down arrow keys it would be awesome!

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2013-10-05T03:27:35Z
Modified: 2014-06-27T14:07:40Z
```

**Comment by taskwarrior** [comment]
_Scott Kostyshak on 2014-06-27T14:07:40Z says:_ 

 Implemented at 716d51d8.
